Configuration/AlCa/python/autoCond.py

```diff
@@ -10,17 +10,19 @@
     # GlobalTag for MC production (p-Pb collisions) with realistic alignment and calibrations for Run1
     'run1_mc_pa'        :   '93X_mcRun1_pA_v0',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run2
-    'run2_design'       :   '93X_mcRun2_design_v0',
+    'run2_design'       :   '93X_mcRun2_design_v1',
```

Comment:
@rekovic you need to rebase this to master

Reply:
or rather, you don't need this file in your PR. autocond from master should suffice. Please remove this.